eng/common/build.ps1

```diff
@@ -50,16 +49,10 @@ function Print-Usage() {
     Write-Host ""
 
     Write-Host "Advanced settings:"
-    Write-Host "  -dogfood                Setup a dogfood environment using the local build"
-    Write-Host "                          This ignores any actions (such as -build or -restore) that were specified"
-    Write-Host "                          If any additional arguments are specified, they will be interpreted as a"
-    Write-Host "                          command to be run in the dogfood context.  If no additional arguments are"
-    Write-Host "                          specified, then the value of the DOTNET_SDK_DOGFOOD_SHELL environment, if"
-    Write-Host "                          it is set, will be used."
     Write-Host "  -projects <value>       Semi-colon delimited list of sln/proj's to build. Globbing is supported (*.sln)"
     Write-Host "  -ci                     Set when running on CI server"
     Write-Host "  -prepareMachine         Prepare machine for CI run"
-    Write-Host "  -fullMSBuild            Test against the desktop version of MSBuild"
+    Write-Host "  -msbuildEngine <value>  Msbuild engine to use to run build ('dotnet', 'vs', or unspecified)."
```

Comment:
From the descriptions, these don't sound like they'll do the same thing. Previously, we would always build with 'dotnet', but -fullMSBuild would instruct the *tests* to run desktop msbuild. It doesn't really matter if we build with full msbuild or not, but we need to make sure that tests will. I think this may cause us to build with desktop msbuild but have tests running core msbuild. That said, I'm going off the description, so I will look at the code to be sure.

Reply:
I see. Then you'll want to keep `-msbuildEngine dotnet`. I'm working on another change that will allow to select the test runtime `/p:TestRuntime=dotnet|full|mono`

Reply:
Filed https://github.com/dotnet/sdk/issues/2589 after we discussed this offline. I will take a look at fixing it.